src/References.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components';
+import firebase from 'firebase/app';
+import { Data } from './Data';
+import { device } from './device';
+import { Link } from 'react-router-dom';
+export default function References() {
+    const name = firebase.auth().currentUser ? firebase.auth().currentUser.displayName : "Hello"
```

Comment:
I see you using `firebase.auth().currentUser` a lot.
You can simplify this by storing the value in a variable like
`const isloggedIn = firebase.auth().currentUser`
Then you can use the `isLoggedIn` check throughout this page